docs/flatpak-metadata/io.github.lavenderdotpet.LibreQuake.metainfo.xml

```diff
@@ -27,17 +27,21 @@
     <p>This release is bundled with the quakespasm sourceport which is released under the
       GPL-2.0 license. See the Github page for details about used licenses.</p>
   </description>
+  <branding>
+    <color type="primary" scheme_preference="light">#c8b2ff</color>
+    <color type="primary" scheme_preference="dark">#241f31</color>
+  </branding>
   <url type="homepage">https://librequake.queer.sh/</url>
   <url type="bugtracker">https://github.com/lavenderdotpet/LibreQuake/issues</url>
   <url type="contribute">https://github.com/lavenderdotpet/LibreQuake</url>
   <screenshots>
     <screenshot type="default">
       <caption>LibreQuake</caption>
-      <image type="source">https://librequake.queer.sh/media/flatpak/storepage04.png</image>
+      <image type="source">https://librequake.queer.sh/media/flatpak/storepage02.png</image>
```

Comment:
gonna have to wait a second till I'm at my computer 

I changed swapped storepage 1 & 2 the other day to try to do it with out updating the flatpak

Reply:
https://github.com/LibreQuake/LibreQuake-Website/commit/54d25358caf7fa11434047beee5c658e78347189

Reply:
Hehe, okay first of all as you can see I didn't order it 1, 2, 3, 4, 5 and secondly the screenshots only get updated during  build. (During build they get mirrored to dl.flathub.org).

So maybe look up whats currently on the website and then give me the exact order you would like. Then I update this here accordingly.

Reply:
ill just undo the change I did on the website first 
then yeah I'll figure out what order

Reply:
its been reverted now
https://github.com/LibreQuake/LibreQuake-Website/commit/cec76212e4278cfba87b0e6932b3943ddeef1583

order for the screen shots
2 → 4 → 6 → 5 → 3 → 1
![image](https://github.com/user-attachments/assets/69965675-e8fc-40da-8263-12716667d086)

side note im thinking eventually we should replace the first one with one based on this but would need to be edited for the correct aspect ratio

![image](https://github.com/user-attachments/assets/b922c5f8-1f7c-48a1-9c6b-d38f6b61b3bb)

Reply:
I have implemented the changes